cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureResourceConnector.java

```diff
@@ -346,6 +340,16 @@ private void checkTemplateDeployment(List<CloudResourceStatus> result, AzureClie
         return resourcesToRemove;
     }
 
+    private void collectAttachedDisks(Map<String, Object> resourcesToRemove, AuthenticatedContext ac, CloudStack stack, AzureCredentialView azureCredentialView,
```

Comment:
so it's a bit misleading, I would expect here to have a collection of disks returned.
instead we add 1 item to a map. so I think this should be renamed, like `addAttachedDiskStorageForRemoval`, or refactor to return the name and call sth like `findAttachedDiskStorageName`